datasets/lt/illegal_gambling/crawler.py

```diff
@@ -0,0 +1,75 @@
+import re
+from typing import Generator, Dict, Tuple
+
+from normality import collapse_spaces
+from zavod import Context
+
+
+def parse_table(
+    table,
+    skiprows=2,
+    headers=["Number", "Name", "Domain", "Contacts", "Court information"],
```

Comment:
I moved this hardcoded list to the top of the module, just a bit of best practice. We also tend to use snake_case, all lower case, for row keys normally - could you update to the same?

Reply:
Did you make this commits in this branch? I didn't find them

Reply:
here https://github.com/opensanctions/opensanctions/commit/0611e60deabb93502cd7d3254383af19cde29dac - sorry that was unclear

Reply:
Do you want me to put these commits in this PR or you'll merge this branch with the https://github.com/opensanctions/opensanctions/pull/636?

Reply:
Please copy the changes or cherry-pick https://github.com/opensanctions/opensanctions/commit/0611e60deabb93502cd7d3254383af19cde29dac
Thanks!

Reply:
Done